sky/clouds/local.py

```diff
@@ -38,6 +39,8 @@ class Local(clouds.Cloud):
             ('Docker image is not supported in Local. '
              'You can try running docker command inside the '
              '`run` section in task.yaml.'),
+        clouds.CloudImplementationFeatures.CUSTOM_DISK_TIER:
+            ('Custom disk tier is not supported for Local.'),
```

Comment:
Why do we set both this and make `check_disk_tier_enabled` possible for `BEST`?

Reply:
It disable `sky launch --disk-tier low <local_cluster>`